diesel_tests/tests/insert.rs

```diff
@@ -214,3 +214,41 @@ fn insert_empty_slice_with_returning() {
     assert_eq!(Ok(None), insert_one.optional());
     assert_eq!(Ok(vec![]), insert_all);
 }
+
+#[test]
+#[cfg(not(feature = "mysql"))]
```

Comment:
We're dropping users and creating it again which would commit the transaction IIRC